src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,16 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Todos = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Todos> = ({ todos }) => (
+  <section className="TodoList">
+
+    {todos.map(todo => (
+      <TodoInfo todo={todo} />
```

Comment:
you have to add `key` property for displaying array.
```suggestion
      <TodoInfo todo={todo} key={todo.id} />
```